packages/libs/src/sdk/api/dashboard-wallet-users/DashboardWalletUsersApi.ts

```diff
@@ -0,0 +1,85 @@
+import type { AuthService, EntityManagerService } from '../../services'
+import {
+  Action,
+  AdvancedOptions,
+  EntityType
+} from '../../services/EntityManager/types'
+import { parseParams } from '../../utils/parseParams'
+
+import {
+  CreateDashboardWalletUser,
+  CreateDashboardWalletUserRequest,
+  DeleteDashboardWalletUserRequest,
+  DeleteDashboardWalletUserSchema
+} from './types'
+
+// TODO(nkang): Extend generated dashboard wallet users API
```

Comment:
You could generate it from staging or local if you want to include that in this pr